### ISSUE TITLE
AVT 1 - Carbon X Search components DAP violation using both examples

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#1831]
## Environment
macOS Mojave version 10.14.2
Chrome version 72.0.3626.96 (Official Build) (64-bit)

## Description
Run DAP against both examples on the same Web page. See screenshot for violations.
<img width="991" alt="screen shot 2019-02-15 at 7 49 12 am" src="https://user-images.githubusercontent.com/21676914/52861611-9cab3780-30f8-11e9-827c-f08b1a9dd60a.png">

Note: This can be corrected by fixing the DAP errors or providing only one example per Web page. It is also recommended that the examples be fully functional.

**Comment by dakahn** [comment]
We're working on better surfacing our a11y test results as providing screen reader examples as spec in our docs. Closing this now since it's specific to our example components and not an actual implementation
